### ISSUE TITLE
Org-drill shows blank question

### ISSUE THREAD

**Issue by Andre0991** [syl20bnr/spacemacs#3238]
Steps to reproduce:

1 - customise the org-modules, enabling "drill"
2 - save the following file:
```
* level 1
** Question :drill:
question 1
*** Answer
ans
** Question :drill:
question 2
*** Answer
ans
** Question :drill:
question 3
*** Answer
ans
```

3 - run org-drill on its buffer and always press 0 when the score is asked. Eventually a blank quesition  shows up.

**Comment by robbyoconnor** [comment]
Thank you for your report. 

This should be [reported upstream](https://github.com/hhirsch/org-drill/issues/new).  It is not included in spacemacs by default -- that said, make a PR to add it, sounds like a nice addition. It's not a problem which we can address.

In the future please read the [contribution guidelines](https://github.com/robbyoconnor/spacemacs/blob/master/CONTRIBUTE.md) prior to writing bug reports as well :stuck_out_tongue:

**Comment by Andre0991** [comment]
Hi @robbyoconnor , thanks for you answer.

Org-drill is included in Spacemacs, but not activated by default (it is packaged in org-plus-contrib, which was added in the last update).

The thing is that this is a Spacemacs specific issue. So I think org-drill's maintainer can't help much.

**Comment by robbyoconnor** [comment]
I'm going to add it.

**Comment by TheBB** [comment]
> The thing is that this is a Spacemacs specific issue.

Have you tried in `emacs -Q`? From what I remember from yesterday this was never done. It could be Spacemacs configuration, but I don't know how you can know that without trying.

**Comment by TheBB** [comment]
But even if it is, it might be best to contact upstream since they can help us or you fix this problem. Org-drill is a bit obscure.

**Comment by Andre0991** [comment]
Sorry @TheBB and @robbyoconnor .
I used to use org-drill very frequently and it used to work fine, so I thought it had to do with the addition of org-plus-contrib. I thought that it was a too basic issue to be unnoticed.

However, I tested on a fresh Emacs installation and I get the same bug.
In fact, it was reported here by another user:

https://bitbucket.org/eeeickythump/org-drill/issues/30/random-blank-buffer-2

Thanks and sorry again.
